src/parser-postcss.js

```diff
@@ -27,9 +27,10 @@ function parseValueNodes(nodes) {
   const commaGroupStack = [commaGroup];
 
   for (let i = 0; i < nodes.length; ++i) {
-    if (nodes[i].type === "paren" && nodes[i].value === "(") {
+    const node = nodes[i];
```

Comment:
Hit having to look into the array when debugging. Easier when it's referenced like this. Kept the change, cuz why not 🙂